server/sonar-server/src/main/java/org/sonar/server/settings/ws/ValuesAction.java

```diff
@@ -143,10 +142,6 @@ private void checkAdminPermission(Optional<ComponentDto> component) {
   }
 
   private List<Setting> loadSettings(DbSession dbSession, Optional<ComponentDto> component, Set<String> keys) {
-    if (keys.isEmpty()) {
```

Comment:
Héhé, I knew you'll ask :).
In fact when looking at the coverage of the class, I couldn't find a way to cover this line : it's because when the "keys" parameter is empty, it's failing. 
So these 3 lines are useless.